src/apikey/lib/controllers/manage.js

```diff
@@ -30,6 +42,18 @@ const createApiKey = async (name, access, attributes, allowed) => {
   return id;
 };
 
+/**
+ * Update an apikey on redis according to its key and its configuration.
+ *
+ * @param {String} id - Key of apikey.
+ * @param {String} name - Name of apikey.
+ * @param {Array<String>} access - Names of the services to which the key has access.
+ * Only accept graphql and enrich.
+ * @param {Array<String>} attributes - Names of the unpaywall attributes
+ * to which the key has access. Only accept attributes from ./attributes.js.
+ * @param {Boolean} allowed - Indicates if the key is authorized or not.
+ * @returns {Object} Config of apikey.
```

Comment:
You could precise the keys of Object (cf. https://jsdoc.app/tags-param.html)

(Not a request change)